app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Admin::OrdersController < ApplicationController
+  respond_to :html, :json
+  def index
+    @orders = Order.all
+    @statuses = @orders.map {|order| order.status}.uniq
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:status` as an argument to `map` instead of a block.